src/OrderFormAcceptanceTests.Steps/Steps/CatalogueSolutions.cs

```diff
@@ -47,6 +47,15 @@ public void GivenThereIsNoCatalogueSolutionInTheOrder()
             searchedOrderItem.Should().BeEmpty();
         }
 
+        [Given(@"a supplier which has a catalogue soltution with only one list price was chosen")]
+        public void GivenASupplierWhichHasACatalogueSoltutionWithOnlyOneListPriceWasChosen()
+        {
+            var order = (Order)Context["CreatedOrder"];
+            order.SupplierId = 100001;
```

Comment:
Can this be grabbed from the db rather than strongly typed?

Reply:
which would you prefer, use the Id to get the name, or use the name to get the Id. Or are you wanting a query which asks for the all suppliers which have a catalogue solution with only one price and then takes the first one?

Reply:
The latter I think. Take a look at the query that Ruth added and change the Order By direction and you should get a single solution supplier

Reply:
Where is that, in her current PR or already in the code somewhere?

Reply:
Current PR